command/server/seal/server_seal_awskms.go

```diff
@@ -11,11 +11,20 @@ import (
 	"github.com/hashicorp/vault/vault/seal"
 )
 
-var getAWSKMSFunc = func(opts *wrapping.WrapperOptions) *awskms.Wrapper { return awskms.NewWrapper(opts) }
+var getAWSKMSFunc = func(opts *wrapping.WrapperOptions, config map[string]string) (wrapping.Wrapper, map[string]string, error) {
+	kms := awskms.NewWrapper(nil)
+	kmsInfo, err := kms.SetConfig(config)
+	if err != nil {
```

Comment:
This is duplicated below; I think just always return kms, kmsInfo, err and let the below check deal with it?

Reply:
yeah. Fixed.